### PR TITLE
Jenkins webhook integration

### PR DESCRIPTION
**Pull request by christian-kreuzberger-dtx**
# This PR

- Adds an example of integration Jenkins with the webhook
- as well as the backlink from Jenkins to Keptn, integrating with the Keptn Jenkins Library (optional)

Preview: https://deploy-preview-952--keptn.netlify.app/docs/0.10.x/integrations/webhooks/jenkins/

### CODE REVIEW

File: content/docs/0.10.x/integrations/webhooks/jenkins/index.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+---
+
+With a Jenkins integration, you can call existing pipelines from Keptn. In addition, with the optional [keptn-jenkins-library](https://github.com/keptn-sandbox/keptn-jenkins-library/)
+you can even provide information back to Keptn (e.g., `test.finished` with `result=fail`).
+
+*Note*: The instructions on this page assume that you do **not** have `jmeter-service` installed.
+
+## Configure Jenkins
+
+*Note*: Keptn needs to be able to reach your Jenkins installation (either directly or via a proxy). This goes way beyond our documentation, therefore we suggest to reach out to your Jenkins administrator.
```
Comment (thisthat) [inline content/docs/0.10.x/integrations/webhooks/jenkins/index.md:15]:
```suggestion
*Note*: Keptn needs to reach your Jenkins installation (either directly or via a proxy). This goes way beyond our documentation, therefore we suggest reaching out to your Jenkins administrator.
```

File: content/docs/0.10.x/integrations/webhooks/jenkins/index.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+  link="./assets/jenkins-trigger-builds-remotely.png"
+  caption="Enable Trigger builds remotely on Jenkins"
+  width="700px">}}
+
+## Store sensitive data in a secret
+
+To secure the sensitive data of your Jenkins webhook URL, a secret needs to be created:
+
+* Go to **Uniform page** > **Secret** and click the **Add Secret**
+* Enter a *name* (e.g., `jenkins-secret`) and select `keptn-webhook-service` for the secret *scope*
```
Comment (thisthat) [inline content/docs/0.10.x/integrations/webhooks/jenkins/index.md:33]: maybe 
```suggestion
* Enter a *name* (e.g., `jenkins-secret`) and select `keptn-webhook-service` as a secret *scope*
```

File: content/docs/0.10.x/integrations/webhooks/jenkins/index.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+* Copy-paste the authentication token from the previous step into the *value* field:
+
+  {{< popup_image
+  link="./assets/jenkins-uniform-secret.png"
+  caption="Create a secret for storing the Jenkins authentication token"
+  width="700px">}}
+
+
+
+## Set up Jenkins integration
```
Comment (thisthat) [inline content/docs/0.10.x/integrations/webhooks/jenkins/index.md:44]:
```suggestion
## Set up Jenkins integration via WebHook
```

File: content/docs/0.10.x/integrations/webhooks/jenkins/index.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+
+  {{< popup_image
+  link="./assets/jenkins-webhook-subscription.png"
+  caption="Create a secret for storing the Jenkins authentication token"
+  width="700px">}}
+
+
+* Finally, click **Create subscription** to save and enable the webhook for your Slack integration.
+
+With those steps done you are triggering a Jenkins Pipeline whenever a `test.triggered` event is sent from Keptn.
```
Comment (thisthat) [inline content/docs/0.10.x/integrations/webhooks/jenkins/index.md:69]:
```suggestion
With those steps done, you are triggering a Jenkins Pipeline whenever a `test.triggered` event is sent from Keptn.
```

**Review by thisthat — APPROVED** [review]

**Comment by christian-kreuzberger-dtx** [comment]
I have added the suggest changes.

**Pull request merged**